js/index.native.js

```diff
@@ -0,0 +1,160 @@
+/**
```

Comment:
Can we do the JS file names the other way around so we have `index.js` and `index.web.js`? According to the [docs](https://github.com/necolas/react-native-web/blob/master/docs/guides/multi-platform-apps.md#web-specific-code) that's the preferred way to do it it.

I don't have any experience with react-native-web though, so if there's a reason to do it this way I'm happy to go with it.

Reply:
I think `index.js` and `index.native.js` is ok, since we are sure that `metro` will use `.native.js` over `.js`, while we are not sure that webpack config in web project will prever `web.js` over `.js`.

Reply:
Yeah, webpack requires extra configuration to enable `.web.js`

Reply:
Ok great! That makes sense 👍